pkg/simple/client/devops/pipeline.go

```diff
@@ -81,6 +81,7 @@ type Pipeline struct {
 // GetPipeBranchRun & SearchPipelineRuns
 type PipelineRunList struct {
 	Items []PipelineRun `json:"items"`
+	Total int           `json:"total_count"`
```

Comment:
We are switching to camelCase variable name instead of snake case, I think `total`  or `totalItems`(which is same with other API) is better here.

Reply:
fixed 
/cc @harrisonliu5